pkg/actuators/machine/reconciler.go

```diff
@@ -187,7 +187,30 @@ func (r *Reconciler) update() error {
 
 	klog.Infof("Updated machine %s", r.machine.Name)
 
-	r.machineScope.setProviderStatus(newestInstance, conditionSuccess())
+	vpc, err := r.awsClient.DescribeVpcs(&ec2.DescribeVpcsInput{
+		VpcIds: []*string{newestInstance.VpcId},
+	})
+	if err != nil {
+		klog.Errorf("%s: error describing vpc: %v", r.machine.Name, err)
+		return err
+	}
+
+	dhcp, err := r.awsClient.DescribeDHCPOptions(&ec2.DescribeDhcpOptionsInput{
```

Comment:
Is this information able to change over the life of the VPC?  If not, maybe we could stuff the domainName in machine's status and only retrieve this if it's missing.

Reply:
DNS Settings can be changed at any time in the VPC lifecycle, so I think it makes sense for this to be done on `update` to check if they've changed (docs: https://docs.aws.amazon.com/vpc/latest/userguide/vpc-dns.html#vpc-dns-updating)

Reply:
Yeah, probably is the safest solution.  Just feels like more API quota exhaustion.  If you have lots of machines, that's two extra API calls per reconcile that could really add up.